blocks/columns/columns.css

```diff
@@ -531,3 +547,56 @@ div.img-col > video {
     width: 469px;
   }
 }
+
+/* ESG View */
+.columns.esg-view em a {
+  display: inline-flex;
+  width: 100%;
+  font-size: 1.25rem;
+  color: var(--secondary);
+  border: 1px solid var(--secondary);
+  background-color: var(--white);
+  padding: 1rem 0;
+  font-weight: 700;
```

Comment:
I think this is present in `styles.css` as a variable

Reply:
Fixed it.